core/src/space/hypeo/mankomania/StageFactory.java

```diff
@@ -197,6 +199,12 @@ public void clicked(InputEvent event, float x, float y) {
             }
         });
 
+        host.addListener(new ClickListener() {
```

Comment:
change this to use a button dedicated to it, not the host button used by network-component. (use a new one)